api_spec.yaml

```diff
@@ -3246,9 +3246,204 @@ paths:
+        
+        '500':
+          $ref: '#/components/responses/500'
+    get:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Get ThirdParty Provider Config
+      operationId: tenantConfigGET
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+        - name: supertokensTenantId
+          in: query
+          schema:
+            type: string
+        - name: thirdPartyId
+          in: query
+          schema:
+            type: string
+      responses:
+        200:
+          description: Config
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: object
+                    properties:
+                      status:
+                        $ref: '#/components/schemas/statusOK'
+                      config:
+                        $ref: '#/components/schemas/thirdPartyProviderConfig'
+                  - type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: ['CONFIG_NOT_FOUND']
+        '400':
+          $ref: '#/components/responses/400'
+        
+        '401':
+          $ref: '#/components/responses/401'
+        
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/config/thirdparty/provider/remove:
+    post:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Delete ThirdParty Tenant Config
+      operationId: deleteTenantConfigPOST
+      parameters:
+        - $ref: '#/components/parameters/multitenancyRid'
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                thirdPartyId:
+                  $ref: '#/components/schemas/thirdPartyId'
+                supertokensTenantId:
+                  $ref: '#/components/schemas/supertokensTenantId'
+
+      responses:
+        200:
+          description: Redirect URL
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    example: OK
+                  didConfigExist:
+                    type: boolean
+        '400':
+          $ref: '#/components/responses/400'
+        
+        '401':
+          $ref: '#/components/responses/401'
+        
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+  /recipe/multitenancy/config/thirdparty/provider/list:
```

Comment:
different input cases:
- only thirdPartyId given -> give for that thirdPartyId across all tenants
- only tenantId given -> give all providers for that tenantId
- none given -> treat as if tenantId is `defaultTenantId`
- both given -> throw an error (400 status code)

Reply:
Done, but instead of 400, return empty array or single element. since we have removed the GET API.